core/services/eth/models.go

```diff
@@ -43,16 +43,20 @@ func NewHead(number *big.Int, blockHash common.Hash, parentHash common.Hash, tim
 	}
 }
 
+// Copy returns a deep copy, excluding Parent.
```

Comment:
This may go away now (or just to tests at least), but if not I'll document better.
It's just meant to avoid racing for the shared `big.Int`s from doing a plain struct copy.

Reply:
Removed